concourse/pipelines/deploy.yml

```diff
@@ -21,17 +21,10 @@ resources:
     icon: github
     name: govuk-infrastructure
     source: &govuk-infrastructure-source
-      branch: main
+      branch: rails_assets_uploads
```

Comment:
This will need setting back before you merge.

Reply:
yes, doing a sanity check first in the pipeline :-)